packages/web-components/src/components/Grid/components/GridItem/GridItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { clsx } from "clsx";
+import type { ComponentChildren, FunctionComponent } from "preact";
+import type { HTMLAttributes } from "preact/compat";
+import styles from "./GridItem.module.scss";
+import { DragHandle } from "./components";
+
+export interface GridItemProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Child elements to render within the grid item.
+   */
+  children: ComponentChildren;
```

Comment:
`FunctionalComponent` already includes `children` property so no need to define it once more